src/main/java/io/vertx/ext/asyncsql/impl/BaseSQLClient.java

```diff
@@ -99,8 +97,6 @@ protected Configuration getConnectionConfiguration(
     String defaultUser,
     String defaultPassword,
     String defaultCharset,
-    long defaultConnectTimeout,
```

Comment:
Note that these parameters should be moved to the pool configuration. I didn't find where it is configured.